src/simplicial-hott/04a-jorthogonality.rzk.md

```diff
@@ -0,0 +1,126 @@
+# 4a. Right orthogonal fibrations
+
+This is a literate `rzk` file:
+
+```rzk
+#lang rzk-1
+```
+
+For every shape inclusion `ϕ ⊂ ψ`,
+we obtain a fibrancy condition for a map `α : A' → A`
+in terms of unique extension along `ϕ ⊂ ψ`.
+This is a relative version of unique extension along `ϕ ⊂ ψ`.
+
+We say that `α : A' → A` is _j-orthogonal_ to the shape inclusion `ϕ ⊂ ψ`,
+or a right orthogonal fibration with respect to `ϕ ⊂ ψ`,
+if the square
+```
+(ψ → A') → (ψ → A)
+
+   ↓          ↓
+
+(ϕ → A') → (ϕ → A)
+```
+is homotopy cartesian.
+
+```rzk title="BW23, Section 3"
+#def is-j-orthogonal-to-shape
+  ( I : CUBE)
+  ( ψ : I → TOPE )
+  ( ϕ : ψ → TOPE)
+  ( A' A : U)
+  ( α : A' → A)
+  : U
+  :=
+    is-homotopy-cartesian
+      ( ϕ → A' ) ( \ σ' → (t : ψ) → A'[ϕ t ↦ σ' t])
+      ( ϕ → A ) ( \ σ → (t : ψ) → A[ϕ t ↦ σ t])
```

Comment:
I think we have a space before `[...]` normally:

```suggestion
      ( ϕ → A' ) ( \ σ' → (t : ψ) → A' [ϕ t ↦ σ' t])
      ( ϕ → A ) ( \ σ → (t : ψ) → A [ϕ t ↦ σ t])
```

Reply:
Ah sorry. I didn't realize.